content/influxdb/v2.0/tools/google-data-studio.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Use the Google Data Studio connector
+description:
+menu:
+  influxdb_2_0:
+    parent: Tools & integrations
+weight: 102
+influxdb/v2.0/tags: [google]
+---
+
+The [InfluxDB Google Data Studio connector](https://datastudio.google.com/u/0/datasources/create?connectorId=AKfycbwhJChhmMypQvNlihgRJMAhCb8gaM3ii9oUNWlW_Cp2PbJSfqeHfPyjNVp15iy9ltCs) lets users create reports and dashboards in Google Data Studio with datasets from InfluxDB v2.
+
+### Add the InfluxDB Connector to Data Studio
+
+1. Add the [InfluxDB Connector data source](https://datastudio.google.com/u/0/datasources/create?connectorId=AKfycbwhJChhmMypQvNlihgRJMAhCb8gaM3ii9oUNWlW_Cp2PbJSfqeHfPyjNVp15iy9ltCs).
+2. Enter the following connection details:
+  - `InfluxDB URL`: Your [InfluxDB 2.0 instance URL](/influxdb/v2.0/reference/urls/).
+  - `Token`: Add an [authentication token](/v2.0/security/tokens/create-token/).
+  - `Organization`: Your [organization ID](/influxdb/v2.0/organizations/view-orgs/#view-your-organization-id).
+  - `Bucket`: Your [bucket ID](/influxdb/v2.0/organizations/buckets/view-buckets/#view-a-list-of-bucket).
```

Comment:
This is auto populated via a drop down when you enter the url, token, and org above

Reply:
it is also the name of the bucket, not the id